packages/uniorg-parse/src/parser.ts

```diff
@@ -425,7 +426,8 @@ class Parser {
       if (objectBegin !== prevEnd) {
         // parse text before object
         const value = this.r.substring(prevEnd, objectBegin);
-        objects.push(u('text', { value }));
+        const end = prevEnd + value.length;
+        objects.push(u('text', { value, ...this.getPosition(prevEnd, end) }));
```

Comment:
nit: `end === objectBegin`
```suggestion
        objects.push(u('text', { value, ...this.getPosition(prevEnd, objectBegin) }));
```